src/utils/permalinks/Permalinks.ts

```diff
@@ -269,26 +269,26 @@ export class RoomPermalinkCreator {
     };
 }
 
-export function makeGenericPermalink(entityId: string): string {
-    return getPermalinkConstructor().forEntity(entityId);
+export function makeGenericPermalink(entityId: string, ispill = false): string {
```

Comment:
Could the param follow the code style & be camelCased to isPill please? I think I read it asm, "I spill" the first time, and that's in the context of this PR.

Also this function could really benefit from some doc now it's not just a simple single-parameter function, as it's not really obvious how the isPill parameter changes the function's behaviour.

Reply:
Where/How should I document that? A separate file in docs/ seems a bit overkill...
Just as comments in the source?

Reply:
Yeah, a jsdoc comment is perfect (eg. https://github.com/matrix-org/matrix-react-sdk/blob/a3240242e8f581512570badd8c39f57eb124d540/src/utils/permalinks/Permalinks.ts#L305)